ui/widget-expressions-variables.md

```diff
@@ -56,18 +56,37 @@ This is useful when "validating" a variable, e.g. send the variable value as com
 
 The `@` symbol can be used in front of an Item name string as a shortcut to the `displayState` from the `items` dictionary with a fallback to the raw state:
 
-```js
+```javascript
 footer: =@'Switch1'
 ```
 
 is the same as
 
-```js
+```javascript
 footer: =items['Switch1'].displayState || items['Switch1'].state
 ```
 
 Similarly, `@@` can be used as a shortcut for just the Item state.
 
+The hashtag `#` symbol is a shortcut to the `numericState` with no fallback:
+
+```javascript
+footer: =#'Temperature1'
+```
+
+is the same as
+
+```javascript
+footer: =items['Temperature1'].numericState
+```
+
+These shortcuts have two major benefits over directly accessing `displayState`, `state` and `numericState`:
+Expressions become shorter when they are used, and if the Item name is a prop which is undefined, these shortcuts avoid that a request with Item name `undefined` is sent to the server, which would cause this log message:
```

Comment:
As the suppressing of the warn message is an important feature I would recommend highlighting this in **bold**.